Test/NotCSharp/Python/EASY-FindWordsThatCanBeFormedByCharacters.py

```diff
@@ -0,0 +1,24 @@
+#https://leetcode.com/problems/find-words-that-can-be-formed-by-characters/
+
+class Solution:
+    def countCharacters(self, words: List[str], chars: str) -> int:
+        from collections import Counter
+        char_dic = dict()
+        tot=0
+        char_dic = Counter(chars)
+        for word in words:
+            count = 0
+            for ch in word:
+                check = True
+                if ch not in char_dic.keys():
+                    check = False
+                    break
+                elif word.count(ch)>char_dic[ch]:
+                    check = False
+                    break
+                else:
+                    count +=1
+            if check:
+                tot+=count
+
+        return tot
```

Comment:
can you double check, I think there's an indentation error on the last line

Reply:
No indentation error on my end,